packages/ui/src/components/DatePicker/DateInput.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+import { Form, InputProps } from '../Form/Form'
+
+export interface DateInputProps extends InputProps {
+	onDateChange?: (value: string) => void
+}
+
+export function DateInput({ name, onDateChange, placeholder }: DateInputProps) {
+	const [value, setValue] = useState(placeholder)
+	const handleDateInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+		const { value } = event.target
+		setValue(value)
+		onDateChange?.(value)
+	}
+
+	useEffect(
+		function () {
+			setValue(placeholder)
+		},
+		[placeholder],
+	)
+
+	return (
+		<Form.Input
+			name={name}
+			placeholder={placeholder}
+			type="input"
+			color={'n8'}
+			value={value}
+			style={{
+				boxSizing: 'border-box',
+				border: 'none',
+				background: 'none',
+				marginTop: '-5px',
```

Comment:
is this required for the font changes?

Reply:
Not for the font change, It's required for the placement of the text and the borders of the container holding the text. I also made some changes to fix the fonts as well.
@Vadman97